cmd/cloudshell_open/main.go

```diff
@@ -253,43 +253,39 @@ func run(opts runOpts) error {
 		return err
 	}
 
-	exists, err := dockerFileExists(appDir)
-	jibMaven := false
-	if err != nil {
-		return err
-	}
-	if exists {
+	projectEnv := fmt.Sprintf("GOOGLE_CLOUD_PROJECT=%s", project)
+	regionEnv := fmt.Sprintf("GOOGLE_CLOUD_REGION=%s", region)
+	serviceEnv := fmt.Sprintf("K_SERVICE=%s", serviceName)
+	pathEnv := fmt.Sprintf("PATH=%s", os.Getenv("PATH"))
```

Comment:
are the envs 100% the same as other hook? if so potentially worth extracting into a common method.

Reply:
Ah ignore me, it seems like you just moved these above.